.github/workflows/testing.yml

```diff
@@ -42,6 +42,7 @@ jobs:
           pushd ..
           npm install node-gyp
           npm install autogypi
```

Comment:
I think we should be able to drop the gyp for now, btw, I forgot to change this when I dropped the depss.

Reply:
good point

Reply:
It can be removed later, maybe?